Formula/gnome-recipes.rb

```diff
@@ -51,6 +51,12 @@ def install
 
     ENV.prepend_path "PKG_CONFIG_PATH", libexec/"lib/pkgconfig"
 
+    # Add RPATH to libexec in goa-1.0.pc on Linux.
```

Comment:
Can we upstream this?

Reply:
This is specific to Homebrew.  In fact it's something we need to start doing automatically for our `.pc` files on Linux, because they will always be broken when used without our compiler shims, which automatically add the RPATH.  Even if that is done automatically, this is using a non-standard directory so it would be needed regardless.